ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -675,9 +675,9 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "messageJudgeOrLegalAdviser",
-    "Name": "Message judge or legal adviser",
```

Comment:
You will need to update the e2e tests, they rely on the name in the dropdown to start the event.